frontend/src/ui/Layout.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { AppHeader } from './AppHeader';
+import { Box } from '@material-ui/core';
+
+export const Layout: React.FC = ({ children }) => {
+  return (
+    <>
+      <AppHeader />
+      <Box display="flex" alignItems="center" justifyContent="center" p={4}>
```

Comment:
Same point here, should we use .css file instead ?

Reply:
https://github.com/french-tech-homies/jukebox/pull/1#discussion_r347064405